### PR TITLE
allow datasetId on project creation

### PR DESCRIPTION
**Pull request by jean-lucas**
# Pull Request Summary

## Description

- Allows project creation to be linked to an existing dataset

[[sc-747086]](https://app.shortcut.com/scaleai/story/747086)

### CODE REVIEW

File: scaleapi/__init__.py [hunk 1]
```diff
@@ -782,20 +782,29 @@ def create_project(
...
                 Project name
-
             task_type (TaskType):
                 Task Type i.e. `TaskType.ImageAnnotation`
-
             params (Dict):
-                Project parameters to be specificed.
+                Project parameters to be specified.
                 i.e. `{'instruction':'Please label the kittens'}`
+            rapid (bool):
```
Comment (gatli) [inline scaleapi/__init__.py:798]: Two boolean variables that are only valid when xor'd. Great 🙈

Kudos on the doc string though!

**Review by gatli — APPROVED** [review]
LGTM!

**Comment by gatli** [comment]
@jean-lucas I don't have write access so it didn't fulfill checks. I'm asking for it now 🤸‍♂️

**Comment by jean-lucas** [comment]
> @jean-lucas I don't have write access so it didn't fulfill checks. I'm asking for it now 🤸‍♂️

All good 👍  I got an automatic grant from Opal when I requested write access to this repo

**Pull request merged**